examples/xed-asmparse-main.c

```diff
@@ -393,6 +393,30 @@ static xed_uint_t count_nibbles(const char *s)
 
 static char const*  const kmasks[] = { "{K0}","{K1}","{K2}","{K3}","{K4}","{K5}","{K6}","{K7}", 0 };
 
+/* Recognize cases when Xed register names are unconvential or buggy */
+static xed_reg_enum_t parse_special_register_name(const char * s) {
```

Comment:
I think I fixed this already. So hopefully this commit's changes are no longer required.